pkgs/top-level/python-packages.nix

```diff
@@ -3291,8 +3291,6 @@ in {
 
   j2cli = callPackage ../development/python-modules/j2cli { };
 
-  jabberbot = callPackage ../development/python-modules/jabberbot { };
```

Comment:
Why are we not adding aliases for those packages?

Reply:
no.